tests/pom.xml

```diff
@@ -320,6 +320,40 @@
         </plugins>
       </build>
     </profile>
+
+    <profile>
+      <id>jacoco</id>
+      <build>
+        <plugins>
+          <plugin>
+            <groupId>org.jacoco</groupId>
+            <artifactId>jacoco-maven-plugin</artifactId>
+            <version>${jacoco.version}</version>
+            <!-- integration tests shouldn't use PowerMock, so recommended prepare-agent goal can be used, instead of offline-instrumentation -->
+            <executions>
+              <execution>
+                <id>jacoco-unit-instrument-classes</id>
+                <configuration>
+                  <skip>true</skip>
+                </configuration>
+              </execution>
+              <execution>
+                <id>jacoco-unit-restore-instrumented-classes</id>
```

Comment:
Should these this be `jacoco-unit-restore-instrumented-classes` or `jacoco-restore-instrumented-classes`?

Reply:
the `id` is something we can make up.